### PR TITLE
Revert "bug #38063 [FrameworkBundle] generate preload.php in src/ to make opcache.preload predictable"

### LINKED ISSUE #38334: [DependencyInjection] Add possibility to avoid creating of `preload.php`
**Symfony version(s) affected**: `4.4.14` / `5.1.6`

**Description**  

With `4.4.14` / `5.1.6` symfony generate a new file in the `src` called `.preload.php` it was introduced by #38063. To allow a general configuration for hosters to preload a file. As I think its a great feature, but sadly in some setups preload is not possible, this is for example currently the state for sulu. That there still a `.preload.php` is generated currently its missleading devs thinking we are now supporting it, so it would be great to have an option to disable this generation.

**How to reproduce**  

```bash
bin/console cache:clear # generate `src/preload.php`
```

**Possible Solution**  

Add a container parameter to disable this behaviour, or make the path to the `preload` file configurable. Instead of a container parameter a `$_SERVER` or `$_ENV` var could be used.

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38334
| License       | MIT
| Doc PR        | -

As discussed in the linked issue, let's replace this with a recipe:
https://github.com/symfony/recipes/pull/825

TL;DR, these PRs replace `src/.preload.php` (generated) by `config/preload.php` (committed).

### CODE REVIEW

**Review by chalasr — APPROVED** [review]

**Review by yceruto — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.
